src/main/scala/io/opentargets/etl/backend/Configuration.scala

```diff
@@ -84,30 +84,30 @@ object Configuration extends LazyLogging {
   )
 
   case class DiseaseOutput(
```

Comment:
Is your Intellij project configured to use the scalafmt file? Lots of our PRs have different white-space, and it would be good to know why.